### PR TITLE
Add docs generated by sphinx

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

**Comment by dshulyak** [comment]
tox -evenv python setup.py build_sphinx

**Comment by bogdando** [comment]
it throws error: invalid command 'build_sphinx'

File: setup.py [hunk 1]
```diff
@@ -45,6 +45,7 @@ def find_requires():
     zip_safe=False,
     install_requires=find_requires(),
     include_package_data=True,
+    setup_requires=['sphinx'],
```
Comment (pigmej) [inline setup.py]: What's the reason for always requiring sphinx ? To me it should be optional dependency.

Comment (dshulyak) [inline setup.py]: i can either add sphinx requirement to [testenv:venv] in tox or leave it here

Comment (dshulyak) [inline setup.py]: or test-requirements

Comment (pigmej) [inline setup.py]: That would be for sure better than listing it as `setup_requires` :)

**Pull request merged**